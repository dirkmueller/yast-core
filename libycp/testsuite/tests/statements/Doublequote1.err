Parsed:
----------------------------------------------------------------------
{
    // filename: "tests/statements/Doublequote1.ycp"
    return (3 + 4);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // filename: "tests/statements/Doublequote1.ycp"
    return 7;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // block <integer> foo1
    // filename: "tests/statements/Doublequote1.ycp"
    block <integer> foo1 = nil;
    foo1 = { return {
    return 42;
}
; };
    return eval (foo1);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // block <integer> foo1
    // integer foo2
    // filename: "tests/statements/Doublequote1.ycp"
    block <integer> foo1 = nil;
    integer foo2 = 55;
    foo1 = { return {
    return (foo2 + 1);
}
; };
    foo2 = 41;
    return eval (foo1);
}
----------------------------------------------------------------------
