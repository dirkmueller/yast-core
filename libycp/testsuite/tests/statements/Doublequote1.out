(7)
(7)
(42)
(42)
